modules/cloud-function-v2/main.tf

```diff
@@ -24,18 +24,6 @@ locals {
       : null
     )
   )
-  _iam_run_invoker_members = concat(
-    lookup(var.iam, "roles/run.invoker", []),
-    var.trigger_config == null ? [] :
-    var.trigger_config.service_account_create ? ["serviceAccount:${local.trigger_service_account_email}"] : []
```

Comment:
do we really need to remove this? can you share the code that triggered the error?